### DISCUSSION TITLE
What is happening with the UI Tests check?

### DISCUSSION THREAD

**Discussion by ollie-iterators** [primer/react#2739] [Q&A] [answered]
### Description

I notice that the UI Tests check always either says that the plan needs to update or that a certain number of changes need to be accepted as baseline. It seems that this number of changes is going up, so when are these changes going to be accepted.

### Steps to reproduce

1. Go to the Pull Requests.
2. Check on one of the pull requests and look at the UI Tests check
3. It will either say that the plan needs to be updated or that there are _ number of changes to be accepted into baseline

Images:

<img width="992" alt="2023-01-04_09-01-01" src="https://user-images.githubusercontent.com/46904826/210571621-5403d824-3c79-4ffa-8e4c-179b5cb1e9fc.png">
<img width="949" alt="2023-01-04_08-59-18" src="https://user-images.githubusercontent.com/46904826/210571625-ffa214e5-f7bc-4908-8440-a7a246a292c6.png">

### Version

v35.2.0

**Comment by joshblack** [comment, marked as answer]
Hi there @ollie-iterators! 👋 

For background, the UI Tests are run by Chromatic and unfortunately we hit our plan limit in  2022. We're gradually moving over to an in-house solution for VRT with Playwright that will ultimately replace this check if things work out.

I'll try and go through the changes today to accept baselines but I assume this may come up again as the check is not required.

Hope this helps! Let me know if you have any more questions 😄
